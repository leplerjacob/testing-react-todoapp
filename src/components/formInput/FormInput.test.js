import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import toJSON from "enzyme-to-json";
import FormInput from "./FormInput";

Enzyme.configure({ adapter: new Adapter() });

describe("FormInput properties", () => {
  test("renders FormInput component", () => {
    let wrapper = shallow(<FormInput />);

    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});