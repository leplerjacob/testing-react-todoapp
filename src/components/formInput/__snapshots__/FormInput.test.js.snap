// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormInput properties renders FormInput component 1`] = `<div />`;
